valetudo-map-card.js

```diff
@@ -45,12 +48,12 @@ class ValetudoMapCard extends HTMLElement {
     this.cardContainer.appendChild(this.controlContainerStyle);
   };
 
-  shouldDrawMap(state, theme) {
-    return !this.drawingMap && (this.lastUpdatedMap != state.last_updated || this.lastDrawnTheme != theme);
```

Comment:
Okay, I understand why you removed them here, but is lastUpdatedMap and lastDrawnTheme are no longer used at all we should get rid of that variable everywhere

Reply:
done